src/emailservice/.dockerignore

```diff
@@ -0,0 +1,2 @@
+Dockerfile
```

Comment:
Maybe we do not need the `.dockerignore` here.
I've added the `Dockerfile`s and the `README`s in the root `.dockerignore`, as all the docker compose build commands should be executed from there.
The PR where I do this should be good to go and merged soon:
https://github.com/open-telemetry/opentelemetry-demo/pull/700/files#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5

This is not a blocker though.

Reply:
Yeah, I saw that one, I think it's a good practice in general, because having the dockerfile added to the image will make all caches invalid, doing a full build everytime. 👍